index.html

```diff
@@ -62,6 +62,142 @@ <h2 class="second-header">I'm a Software Developer</h2>
+            </div>
+            <div class="work-card">
+                <p class="work-cards-title">Profesional Art Printing Data</p>
+                <p class="work-cards-description">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard.</p>
+                <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button class="button-card" type="button">See Project</button>
+            </div>
+            <div class="work-card">
+                <p class="work-cards-title">Profesional Art Printing Data</p>
+                <p class="work-cards-description">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard.</p>
+                <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button class="button-card" type="button">See Project</button>
+            </div>
+            <div class="work-card">
+                <p class="work-cards-title">Profesional Art Printing Data</p>
+                <p class="work-cards-description">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard.</p>
+                <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button class="button-card" type="button">See Project</button>
+            </div>
+            <div class="work-card">
+                <p class="work-cards-title">Profesional Art Printing Data</p>
+                <p class="work-cards-description">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard.</p>
+                <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button class="button-card" type="button">See Project</button>
+            </div>
+            <div class="work-card">
+                <p class="work-cards-title">Profesional Art Printing Data</p>
+                <p class="work-cards-description">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard.</p>
+                <ul>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button class="button-card" type="button">See Project</button>
+            </div>
+        </div>
+    </section>
+    <section class="about-me-main-container">
+        <header>
+            <h1 class="about-me-title">About me</h1>
```

Comment:
- [ ] You should use any other `hx` tag but not `h1` to improve your page SEO optimization.

Reply:
@bobb-Rob Thank you very much for your feedback! Have an amazing day.